config/environments/development.rb

```diff
@@ -27,19 +42,17 @@
   # number of complex assets.
   config.assets.debug = true
 
-  # Asset digests allow you to set far-future HTTP expiration dates on all assets,
-  # yet still be able to expire them through the digest params.
-  # config.assets.digest = true
-
-  # Adds additional error checking when serving assets at runtime.
-  # Checks for improperly declared sprockets dependencies.
-  # Raises helpful error messages.
-  config.assets.raise_runtime_errors = true
+  # Suppress logger output for asset requests.
+  config.assets.quiet = true
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
-  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
```

Comment:
__[rubocop]__
* Style/CommentIndentation: Incorrect indentation detected (column 1 instead of 3).
* Style/CommentIndentation: Incorrect indentation detected (column 2 instead of 1).
* Style/CommentIndentation: Incorrect indentation detected (column 3 instead of 2).

<!-- Comment by [SideCI](https://sideci.com) -->